package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "hello-wasm",
+  "name": "wasmwrap",
```

Comment:
Hm, if we decide on a name, we should probably do this a bit more consciously, and e.g. choose one that notes that this package is only meant to be used with Rust.